src/dispatch/plugins/dispatch_slack/fields.py

```diff
@@ -81,6 +106,41 @@ class TimezoneOptions(DispatchEnum):
     utc = "UTC"
 
 
+def relative_date_picker_input(
+    action_id: str = DefaultActionIds.relative_date_picker_input,
+    block_id: str = DefaultBlockIds.relative_date_picker_input,
+    initial_option: dict = None,
+    label: str = "Hour",
+    time_format: str = "%Y-%m-%d %H:%M:%S",
+    **kwargs,
+):
+    """Builds a relative date picker input."""
+    now = datetime.now()
+
+    relative_dates = [
+        {"text": "1 hour", "value": now + timedelta(hours=1)},
```

Comment:
Shouldn't we delay the calculation until creation? I'm thinking of a situation where somebody opens a modal and goes and gets a coffee.